main.rb

```diff
@@ -1,37 +1,54 @@
 require './app'
 
+@persons = []
+@books = []
+@rentals = []
+
 def main
-  app = App.new
-  app.home_page
+  home_page
+end
+
+def home_page
+  puts 'Welcome to Library App!'
+  until list_of_options
+    input = gets.chomp.to_i
+    if input == 7
+      puts 'Thank you for using Library App!'
+      exit
+    end
+
+    option input
+  end
 end
 
 def list_of_options
   puts
   puts 'Please select an option by entering a number:'
-  puts '1. List all persons'
-  puts '2. List all books'
-  puts '3. List all rentals for a given person id'
-  puts '4. Add a person'
-  puts '5. Add a book'
-  puts '6. Add a rental'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
```

Comment:
You have done well resolving the requested changes, however while fixing them, you have accidentally added a bug into the codebase. When I create a teacher, the app breaks as you can see in the below video. Consider fixing this in order for the app to behave correctly.